front/app/components/admin/PostManager/components/FilterSidebar/phases/FilterSidebarPhasesItem.tsx

```diff
@@ -54,7 +55,10 @@ const FilterSidebarPhasesItem = ({
         labelContent={
           <LabelContentWrapper>
             {localize(phase.attributes.title_multiloc)}
-            <CircledPhaseNumber phaseNumber={phaseNumber} />
+            {/* Flex shrink needed to prevent number from becoming squished  */}
+            <Box flexShrink={0}>
```

Comment:
Confused about why it doesn't work when we put flexShrink directly on CircledPhaseNumber's Box 🤔 Do you know?

Reply:
From when I looked online, I think it's possible the `width="26px"` overrides the shrink :thinking: